packages/node_modules/@ciscospark/common/src/constants.js

```diff
@@ -1,49 +1,55 @@
-export const WEBEX_EVENT = {
-  TEAMS_ACTIVITY: 'event:conversation.activity'
-};
-
-export const WEBEX_SPACE_TYPE = {
-  DIRECT: 'direct',
-  GROUP: 'group'
-};
-
-export const API_ACTIVITY_FIELD = {
-  ACTOR: 'actor',
-  OBJECT: 'object',
-  TARGET: 'target'
-};
-
-export const API_ACTIVITY_VERB = {
-  ACKNOWLEDGE: 'acknowledge',
-  POST: 'post',
-  SHARE: 'share',
-  ADD: 'add',
-  LEAVE: 'leave',
-  ADD_MODERATOR: 'assignModerator',
-  REMOVE_MODERATOR: 'unassignModerator',
-  HIDE: 'hide'
-};
-
 export const SDK_EVENT = {
-  CREATED: 'created',
-  DELETED: 'deleted',
-  UPDATED: 'updated',
-  SEEN: 'seen'
-};
-
-export const SDK_EVENT_OWNER = {
-  CREATOR: 'creator',
-  ORG: 'org'
-};
-
-export const SDK_EVENT_STATUS = {
-  ACTIVE: 'active',
-  DISABLED: 'disabled'
-};
-
-export const SDK_EVENT_RESOURCE = {
-  MEMBERSHIPS: 'memberships',
-  MESSAGES: 'messages'
+  FROM_WEBEX: {
```

Comment:
Let's use `INTERNAL` to keep it consistent with the plugin names, e.g., `internal-plugin-encryption`. Sound good?

Reply:
Agree see commit ea3898f8de7a0270f00ea41760802cf260449a8d